SConstruct

```diff
@@ -310,6 +310,12 @@ else:
     )
     env.Append(CFLAGS=["-std=c11"])
 
+if env.get("use_mingw", False):
```

Comment:
```suggestion
if env.get("use_llvm", False):
```

This will not work with normal MinGW/GCC distributions, since there's no `timespec_get` and it's not using UCRT. Library will build, but Godot won't (linking will fail).

Or maybe it with adding a separate `use_ucrt` flag, since at least for x86-64, MinGW/clang can use both `UCRT` and `msvcrt`).

Reply:
I do not have a strong opinion I can change it to check for use_ucrt if preferable.